### PR TITLE
feat: infra init

### PR DESCRIPTION
**Pull request by dixitaniket**
closes: #14

### CODE REVIEW

File: infra/go.mod
Comment (adamewozniak) [inline infra/go.mod:1]:
```suggestion
module contract-relayer
```
?

File: infra/provision.go
Comment (adamewozniak) [inline infra/provision.go:51]: Are we spinning up a node here? I thought the goal should be to spin up a relayer to an existing contract quickly

File: infra/provision.go
Comment (adamewozniak) [inline infra/provision.go:57]: This ssh key for any stack can be easily generated by https://github.com/umee-network/umee-infra/blob/main/lib/sshkeygen/ssh.go

File: infra/provision.go
Comment (adamewozniak) [inline infra/provision.go:158]: If we're creating a new wallet to use for each cw relayer we should also generate a password & put it as a secret in the pulumi config. We should consider these wallets funded & production

File: infra/provision.go
Comment (adamewozniak) [inline infra/provision.go:166]: Let's set these as constants

File: infra/provision.go
Comment (adamewozniak) [inline infra/provision.go:181]: might want to move this over to scripts

File: infra/provision.go
Comment (adamewozniak) [inline infra/provision.go:137]: might want to move this over to scripts file

**Review by adamewozniak — COMMENTED** [review]
this is looking good! just a few comments - we want to make sure each stack has a separate password for the wallet backend

**Comment by adamewozniak** [comment]
got eyes on this 👍

File: infra/provision.go
Comment (adamewozniak) [inline infra/provision.go:43]: consts please

File: infra/provision.go
Comment (adamewozniak) [inline infra/provision.go:201]:
```suggestion
	// start relayer daemon, can also be removed as it already exists
```

**Review by adamewozniak — APPROVED** [review]
a couple nitpicks but this looks good to me!
Let's test a release here before merging

**Comment by adamewozniak** [comment]
@dixitaniket please merge this in if your tests have gone through 👍

**Review by zarazan — APPROVED** [review]
Based on my current pulumi knowledge I think this looks great.

**Pull request merged**
